canonical-web-and-design/maas-ui
# feat(ui): right align release notification settings link

## Done

- Update the new release notification to show the settings link on the right.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new release notification (instructions are here: https://github.com/canonical-web-and-design/MAAS-squad/issues/2065#issue-669660220).
- Check that the "See settings" link is aligned right (next to the X).

<img width="1072" alt="Screen Shot 2020-08-26 at 10 18 30 am" src="https://user-images.githubusercontent.com/361637/91240489-823e2980-e785-11ea-9c64-6833e9bf1972.png">
